### ISSUE TITLE
Render RichTextField using LaTex

### ISSUE THREAD

**Issue by eeintech** [donhauser/wagtail-pdf#1]
Following the example on the Wagtail tutorial: https://docs.wagtail.io/en/stable/getting_started/tutorial.html#extend-the-homepage-model

How do you render the body `RichTextField` in the Tex template?

I've tried:
``` latex
{{- page.body -}}
```

and:

``` latex
{{- page.body|richtext -}}
```

to no success.

Thanks!

**Comment by donhauser** [comment]
I've tried to reproduce your problem and found the following problems:
```
{{- page.body -}}
```
does not work (and likely will not work in the future), because you are pasting HTML code into LaTeX, which then causes compile problems because of chars like: `<` and `>`.

On the other hand the following should work out, but the parsing was crashing because of a bug.
```
{{- page.body|richtext -}}
```
In this scenario the `RichTextField` is processed with the function `richtext_as_tex()` and then processed with `SimpleHtmlToLatexParser`, which are both located in *environment.py* (just for reference).

In version 0.1.1 `richtext_as_tex()` tries to access the `__html__` property of the field, but the input is simply a string, causing a failure in my case. I've fixed this in the code and **updated the project to version 0.1.2**, so if you update the wagtail-pdf-view package via `pip install --upgrade`, then the `RichTextField` should work.

I hope this helps, cheers!
